h/templates/activity/search.html.jinja2

```diff
@@ -273,38 +273,25 @@
 
     {{ tags_section() }}
 
-    {# if the group has a concept of members display members and join text #}
-    {% if group.members %} 
-      {{ group_users("Members", group.members, group.creator) }}
-      <section class="search-result-sidebar__section">
-        <h3 class="search-result-sidebar__subtitle">
-          {% trans %}Invite new members{% endtrans %}
-        </h3>
-        <p class="search-result-sidebar__share-link">
-          {% trans %}Sharing the link lets people join this group:{% endtrans %}
-        </p>
-    {# if the group has no concept of members display moderators and share text #}
-    {% else %}
-      {{ group_users("Moderators", group.moderators, group.creator) }}
-      <section class="search-result-sidebar__section">
-        <h3 class="search-result-sidebar__subtitle">
-          {% trans %}Share group{% endtrans %}
-        </h3>
-        <p class="search-result-sidebar__share-link">
-          {% trans %}Sharing the link lets people view this group:{% endtrans %}
-        </p>
-    {% endif %}
-    <div class="group-invite__container js-copy-button">
-      <input class="group-invite__input js-select-onfocus"
-             data-ref="input"
-             value="{{ group.url }}">
-      <button class="group-invite__clipboard-button"
-              data-ref="button"
-              title="Copy to clipboard">
-        {{ svg_icon('copy_to_clipboard', 'group-invite__clipboard-image') }}
-      </button>
-    </div>
-  </section>
+    {{ group_users(group_users_display_info.title, group_users_display_info.members, group.creator) }}
```

Comment:
Can I suggest a couple of changes to `group_users_display_info`?

1. Rename it to `group_users_args`, because it's really just the list of args that will be passed to the `group_users()` macro
2. Add `group.creator` into `group_users_args`, so that _all_ the args that will be passed into `group_users()` are in `group_users_args`
3. Make `group_users_args` a list, rather than a dict. Then you can call `group_users()` with `*` like: `group_users(*group_users_args)`.

Something like this:

```diff
diff --git a/h/templates/activity/search.html.jinja2 b/h/templates/activity/search.html.jinja2
index 6757a8f55..a014cec85 100644
--- a/h/templates/activity/search.html.jinja2
+++ b/h/templates/activity/search.html.jinja2
@@ -273,7 +273,7 @@
 
     {{ tags_section() }}
 
-    {{ group_users(group_users_display_info.title, group_users_display_info.members, group.creator) }}
+    {{ group_users(*group_users_args) }}
     <section class="search-result-sidebar__section">
       <h3 class="search-result-sidebar__subtitle">
         {{ group.share_subtitle }}
diff --git a/h/views/activity.py b/h/views/activity.py
index 427e91dbe..d302c6f84 100644
--- a/h/views/activity.py
+++ b/h/views/activity.py
@@ -155,11 +155,6 @@ class GroupSearchController(SearchController):
             'annotation_count': group_annotation_count,
         }
 
-        result['group_users_display_info'] = {
-            'title': _('Members'),
-            'members': moderators,
-        }
-
         result['group'] = {
             'created': utc_us_style_date(self.group.created),
             'description': self.group.description,
@@ -178,10 +173,13 @@ class GroupSearchController(SearchController):
         if self.group.type == 'private':
             result['group']['share_subtitle'] = _('Invite new members')
             result['group']['share_msg'] = _('Sharing the link lets people join this group:')
-            result['group_users_display_info']['members'] = members
 
-        if self.group.type == 'restricted':
-            result['group_users_display_info']['members'] = members
+        result['group_users_args'] = [
+            _('Members'),
+            moderators if self.group.type == 'open' else members,
+            result['group']['creator'],
+        ]
+
 
         if self.request.has_permission('admin', self.group):
             result['group_edit_url'] = self.request.route_url(
```